src/wait-config.html

```diff
@@ -0,0 +1,60 @@
+<script type="text/javascript">
+    function setWaitAction() {
+        let action = $("#node-input-action").val()        
+        $('#actionImplicit').show()
+        $('#actionPageLoad').show()
+        $('#actionScript').show()
+        if (action === 'get') {
+            $('#actionImplicit').hide()
+            $('#actionPageLoad').hide()
+            $('#actionScript').hide()
+        }
+    }
+
+    RED.nodes.registerType('wait-config', {
+        category: 'Webdriver IO',
+        color: '#a6bbcf',
+        defaults: {
+            name: {value: ''},
+            implicit: {value: ''},
+            pageload: {value: ''},
+            script: {value: ''}
+        },
+        inputs: 1,
+        outputs: 1,
+        icon: 'white-globe.png',
+        label: function() {
+            return this.name || "wait config"
+        },
+        oneditprepare: function() {
+            setWaitAction()
+        }
+    })
+</script>
+
```

Comment:
What does "wait config" mean? What is this node used for, vs the "explicit wait" node?

Reply:
Wait-config is used at the beginning of session to set the global waits for implicit wait for each element to be displayed, Page load wait and script waits. Explicit waits are used specifically for web elements to be displayed/enabled/selected/not displayed.. Explicit waits are used in a flow where we need to add some extra time to wait before we do actions on elements. Web pages with AJAX code will render page/iframe after certain time, So we can wait till the element is displayed and click.

Reply:
Would it be better to call this "implicit-wait-config" then?

Reply:
Sure Rob! Will add the change